content/guides/sync.md

```diff
@@ -68,74 +70,76 @@ This section, together with a [DEFAULT] section, constitutes the full path of av
 	PrefilterConsumerFilePath = /var/lib/ar-consumer/ar-consumer_log_%s.avro
 	PrefilterPoem = poem_sync_%s.out
 	PrefilterPoemNameMapping = poem_name_mapping.cfg
-	TopologyGOCDBGroupOfEndpoints = group_endpoints_%s.avro
-	TopologyGOCDBGroupOfGroups = group_groups_%s.avro
-	TopologyVOGroupOfEndpoints = group_endpoints_%s.avro
-	TopologyVOGroupOfGroups = group_groups_%s.avro
+	TopologyGroupOfEndpoints = group_endpoints_%s.avro
+	TopologyGroupOfGroups = group_groups_%s.avro
 	WeightsGstat = weights_%s.avro
 
 Section lists all the filenames that each component is generating. Directory is purposely omitted because it's implicitly found in next configuration file. Exception is a `PrefilterConsumerFilePath` and `PrefilterPoem` options that tells the `prefilter-egi.py` where to look for its input files. `%s` is a string placeholder that will be replaced by the date timestamp in format `year_month_day`.
 
 ### customer.conf
 
```

Comment:
This section confuses me. In the example below is it supposed that EGI is a customer and NGI another customer?